server/modules/auth/auth.middleware.ts

```
@@ -0,0 +1,135 @@
+  ) {
+    token = req.headers.authorization.split(' ')[1];
+  }
+
+  // If no token is found, deny access
+  if (!token) {
+    return next(new ErrorResponse('Not authorized, no token found', 401));
+  }
+
+  try {
+    // Verify token
+    const decoded: any = jwt.verify(token, process.env.JWT_SECRET!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)